### ISSUE TITLE
Modern Ubuntu

### ISSUE THREAD

**Issue by MattCurryCom** [MicrosoftDocs/windowsserverdocs#2588]
Any with 18.04/18.10?

When I do this for 18.10, it adds repo no problem.  But still doesn't find the pkgs
like blobfuse for example...

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1e7ed059-d77f-6ce0-8501-fc3afd1c8048
* Version Independent ID: f2d249ee-50e4-52b4-4187-954a84a48ec1
* Content: [Linux Software Repository for Microsoft Products](https://docs.microsoft.com/en-us/windows-server/administration/Linux-Package-Repository-for-Microsoft-Software#feedback)
* Content Source: [WindowsServerDocs/administration/Linux-Package-Repository-for-Microsoft-Software.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/Linux-Package-Repository-for-Microsoft-Software.md)
* Service: **na**
* Product: **windows-server-threshold**
* GitHub Login: @szarkos
* Microsoft Alias: **szark**

**Comment by abair34** [comment]
TLDR;  https://packages.microsoft.com/ubuntu/16.04/prod
sometimes is really (for 2019 CTPs for example)
https://packages.microsoft.com/ubuntu/16.04/mssql-server-preview/
so browse to the 16.04 or the 18.04 root and make sure you find the folder you need with dist and pool in it...

I wanted to add polybase to a 16.04 container today for a quick and dirty POC.  Linux repos are browse-able so hit the webpage (because what I'm about to tell you will change when things get out of preview).  For me to add the polybase repo to a docker container running in k8s I had to do the following...

download the .asc file so I can copy it to the container
kubectl cp microsoft.asc <pod name from kubectl get pods>:<folder path we can copy to>
kubectl exec <pod name from kubectl get pods> -i -t -- bash -il
now you're in bash so cd to wherever you copied the asc to
apt-key add microsoft.asc
apt-get update
apt-get install software-properties-common
apt-add-repository https://packages.microsoft.com/ubuntu/16.04/mssql-server-preview
apt-get update
apt-get install mssql-server-polybase

In retro if you're working on a container you probably shouldn't follow my exact steps... just download the .deb and copy it there and install that (you don't want to bloat the container with the things my steps will do).   

HTH
